amazon_price_per_item.user.js

```diff
@@ -58,13 +67,46 @@
       parsePrice: parsePrice,
     };
   }
+
   // If we're not returning exports, run the gm script
   var ID = "gm_amazon_price_per_item";
   var DEBUG = false;
   if (!DEBUG && window.location.search.indexOf("appi=1")) {
     DEBUG = true;
   }
   var STYLE_ESTIMATED = "color:blue;";
+  //var CURRENCY_SIGN = "$";
+  var CURRENCY_MODE='USD'
+
+  var LOCALE=document.documentElement.lang //"en-US"
+  let segments = LOCALE.split('-'), country=segments[0], region=segments[1];
+  //var DECIMAL_SEPARATOR="."
+  var DECIMAL_SEPARATOR= new Intl.NumberFormat(LOCALE).format(1.1).replaceAll('1','')
+  //var THOUSANDS_SEPARATOR=",";
+  var THOUSANDS_SEPARATOR=new Intl.NumberFormat(LOCALE).format(1111).replaceAll('1','')
+  let currencies = {
+    fr: 'EUR',
+    en: 'USD'
+  }
+  CURRENCY_MODE=currencies[country]||'USD'
+
+  let i18n_strings = {
+    fr: {
+        item: 'unité',
+        estimated: function(perItem){ return `Environ ${perItem} par unité`}
+    },
+    en: {
+        item: 'item',
+        estimated: function(perItem){ return `Estimated ${perItem} per item`}
+    }
+  };
+  let i18n=i18n_strings[country]
+  let UNIT_WORD = i18n.item;
+
+  let formatter = new Intl.NumberFormat(LOCALE, {
```

Comment:
This is neat. You've taught me quite a bit about i18n in this MR. I appreciate it and will do my best to keep it updated.